fenix/app/src/main/java/org/mozilla/fenix/translations/TranslationSwitchItem.kt

```diff
@@ -0,0 +1,24 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.translations
+
+/**
+ * TranslationSwitchItem that will appear on Translation screens.
+ *
+ * @property textLabel the text that will appear on the switch.
+ * @property contentDescription for the switch item.
+ * It will be a single contentDescription for the switch and for the text next to the switch.
+ * @property isEnabled if the switch is checked or not.
+ * @property hasDivider if a divider should appear under the switch.
+ * @property onStateChange is Invoked when Switch is being clicked,
+ * therefore the change of checked state is requested.
```

Comment:
```suggestion
 * @property textLabel The text that will appear on the switch item.
 * @property contentDescription The content description for the switch item.
 * It will be a single content description for the switch and for the text next to the switch.
 * @property isChecked Whether the switch is checked or not.
 * @property hasDivider Whether a divider should appear under the switch item.
 * @property onStateChange Invoked when the switch item is clicked,
 * the new checked state is passed into the callback.
```
We should avoid using the param name as the start of the sentence and just write complete sentences. The rendered view of these kDocs as is would look like: `onStateChange - is Invoked ...` when we want to `onStateChange - Invoked`.

Example rendered view:
<img width="561" alt="Screen Shot 2023-10-17 at 11 31 39 AM" src="https://github.com/mozilla-mobile/firefox-android/assets/1190888/36133176-4d08-4b3d-80f5-9df1e4708ef0">